docs/using-pretrained-models.md

```diff
@@ -14,20 +14,44 @@ Utilizing pretrained models can reduce training time and resource usage.
 To download and use models from previous training runs or external sources, use the `pretrained-models` parameter in the training config. The keys in this parameter correspond to the training task `kinds` capable of using pretrained models. This is currently `train-teacher` and `train-backwards`. See [#515](https://github.com/mozilla/firefox-translations-training/issues/515) for `train-student` support.
 
 ```yaml
-pretrained-models:
-  # Continue training a teacher model.
-  train-teacher:
-    urls:
-      - https://firefox-ci-tc.services.mozilla.com/api/queue/v1/task/task-id/artifacts/public/build
-    mode: continue
-    type: default
-
-  # Re-use an existing backwards model from a Google Cloud Storage bucket.
-  train-backwards:
-    urls:
-      - https://storage.googleapis.com/bucket-name/models/ru-en/backward
-    mode: use
-    type: default
+experiment:
+  pretrained-models:
+    # Continue training a teacher model.
+    train-teacher:
+      urls:
+        - https://firefox-ci-tc.services.mozilla.com/api/queue/v1/task/task-id/artifacts/public/build
+      mode: continue
+      type: default
+
+    # Re-use an existing backwards model from a Google Cloud Storage bucket. This must
+    # be the original (non-quantized) student model.
+    train-backwards:
+      urls:
+        - https://storage.googleapis.com/releng-translations-dev/models/en-fi/opusmt/student/
+      mode: use
+      type: default
+```
+
+To find models from older training runs see the `gs://releng-translations-dev/models` bucket.
```

Comment:
Note: the long term storage location for production training runs will be `moz-fx-translations-data--303e-prod-translations-data`. (I think we'll eventually end up migrating all of the data to this bucket and getting rid of `releng-translations-dev`, which was only intended as a short term workaround.)